test/remote_reporter.js

```diff
@@ -25,17 +25,25 @@ import MockLogger from './lib/mock_logger.js';
 import RemoteReporter from '../src/reporters/remote_reporter.js';
 import Tracer from '../src/tracer.js';
 import UDPSender from '../src/reporters/udp_sender.js';
+import MetricsContainer from '../src/metrics/metrics.js';
```

Comment:
where is the "container" name coming from?  If it's simply importing the Metrics class, I think it's less confusing to call it Metrics.